modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/test/js/components/export-form/ExportFormModal.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {act, fireEvent, render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import {ExportFormModal} from 'dynamic-data-mapping-form-web/admin/js/components/export-form/openExportFormModal.es';
+import React from 'react';
+
+import '@testing-library/jest-dom/extend-expect';
+
+const props = {
+	csvExport: 'enabled-with-warning',
+	exportFormInstanceURL: 'https://liferay.com/admin/export_form_instance',
+	fileExtensions: {
+		csv: 'CSV',
+		json: 'JSON',
+		xls: 'XLS',
+		xml: 'XML',
+	},
+	portletNamespace: 'portletNamespace_',
+	spritemap: 'spritemap',
+};
```

Comment:
Prefer to specify properties directly into the component declaration, using only the stricted needed in a test case. It will make the tests easier to understand, maintain and will prevent that a prop changes its value from one test case to another, due to a miss manipulation.

It's ok to create a variable to like `fileExtensions`, but ensure that its value is reassigned on every test execution (it will ensure the test isolation):
```javascript
let fileExtensions;

beforeEach(() => {
	fileExtensions = {csv: 'CSV', json: 'JSON', xls: 'XLS', xml: 'XML'};
});
```